app/lib/builder/views/editor/Page.svelte

```diff
@@ -91,32 +91,32 @@
 			await repair_section_indices()
 		}
 
-		if (sections.length === 0 && page_type_body_sections.length !== 0) {
-			await copy_default_sections()
-		}
-
 		// Adjust indices of existing sections that come after the insertion position
 		const existing_sections = sections.filter((section) => section.index >= position)
 
-		for (const section of existing_sections) {
-			PageSections.update(section.id, { index: section.index + 1 })
-		}
-
-		// Create new section
+		// Create new section with correct index
 		const new_section = PageSections.create({
 			page: page.id,
 			symbol: symbol.id,
 			index: position
 		})
-		await manager.commit()
 
-		// Add to newly added sections for animation
-		newly_added_sections.add(new_section.id)
+		// Copy symbol entries to the new section instance
+		for (const entry of symbol.entries()) {
```

Comment:
Note that using collection accessor methods (one, list, and link methods) inside a callback should be considered a bad practice because they don't guarantee that data is loaded immediately. But in this case it might work because symbol entries are being loaded in the sidebar during symbol preview rendering. Might cause problems at some point though.


`symbol.entries()` can return `undefined` in the case that loading is in progress, so please add a fallback to avoid runtime errors.
```suggestion
		for (const entry of symbol.entries() ?? []) {
```